hw_1/task_4.py

```diff
@@ -0,0 +1,13 @@
+# Задача 8: Требуется определить, можно ли от 
+# шоколадки размером n × m долек отломить k долек, 
+# если разрешается сделать один разлом по прямой 
+# между дольками (то есть разломить шоколадку на два прямоугольника).
+
+# *Пример:*
+
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+
+n, m, k = map(int, (input().split(" ")))
```

Comment:
`split` без аргументов тоже будет делить строку по пробелам, причем даже лучше, так как не будет учитывать количество пробелов
```suggestion
n, m, k = map(int, input().split())
```